### ISSUE TITLE
Configure EIS Freshwater story for the dashboard

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/veda-config#77]
- [ ] Make sure all datasets required have been published
- [ ] Configure all datasets in delta-config ([Content documentation](https://github.com/NASA-IMPACT/delta-config/blob/develop/docs/CONTENT.md) is a good place to start)
- [ ] Configure story content in delta-config
- [ ] Developer opens PR to `develop`
    - [ ] Make sure automated unit tests pass
    - [ ] Developers creating and reviewing PR can check the preview 
<img width="1055" alt="Screen Shot 2022-07-20 at 11 45 36 AM" src="https://user-images.githubusercontent.com/15016780/180025997-16948f9a-cdb8-428f-9bc5-f3d6b31bb8b2.png">

- [ ] Merging to `develop` will trigger to staging environment: [.github/workflows/deploy-staging.yml](https://github.com/NASA-IMPACT/delta-config/blob/develop/.github/workflows/deploy-staging.yml): http://visex.netlify.com/
- [ ] Preview story with story tellers in http://visex.netlify.com/. Do some adhoc testing of other stories and testing.
- [ ] Check in with product owner (Anthony B) about what types of communication should be made before deploying to production.
- [ ] If everything looks good, open PR from `develop` to `main`. Once merged this will deploy to https://www.earthdata.nasa.gov/dashboard/

**Comment by aboydnw** [comment]
Thanks @abarciauskas-bgse this was a super helpful concise layout of the steps. I think it will be useful to iterate on this or include as part of the documentation as we involve more folks in story writing. 

I had been working on updating our VEDA Workflow miro board with some more details on the current data ingestion & configuration workflows. I tried to capture the big steps you have here in the [miro workflow.](https://miro.com/app/board/uXjVOB6JoZM=/?moveToWidget=3458764529696998297&cot=14) Let me know if this captures the workflow well @hanbyul-here @danielfdsilva and if you think it would valuable to review with IMPACT.

@abarciauskas-bgse @anayeaye @alukach @ingalls I also took a first stab at making something similar for [dataset ingestion](https://miro.com/app/board/uXjVOB6JoZM=/?moveToWidget=3458764529696822253&cot=14). Same question to you all, is this accurate and is it worth iterating/reviewing with IMPACT?

Dotted boxes/lines are "where we would like to go" aka the potential workflow after completing some development work in progress (Register STAC API, multi-DAAC support, story configuration UI, etc.)
